analyzers/dataframe/src/myUtils.cc

```diff
@@ -169,10 +195,10 @@ ROOT::VecOps::RVec<float> get_Vertex_d2PV(ROOT::VecOps::RVec<VertexingUtils::FCC
   ROOT::VecOps::RVec<float> result;
   VertexingUtils::FCCAnalysesVertex PV;
   for (auto &p:vertex)
-    if (p.vertex.primary>0) PV=p;
+    if (isPrimaryVtx(p.vertex)) PV=p;
 
   for (auto &p:vertex){
-    if (p.vertex.primary>0) result.push_back(0);
+    if (isPrimaryVtx(p.vertex)) result.push_back(0);
```

Comment:
These technically do not test for equality but from the function name, I deduced that *primary vertex* is the desired quantity here.

Reply:
This seems to be part of some performance example(?)
https://github.com/HEP-FCC/FCCAnalyses/blob/ff50f2541693c369e49790ae3249bbb4e5d13997/examples/FCCee/vertex_perf/analysis.py#L84-L92

As well as some flavor analyses:
- flavour/BuBc2TauNu
- flavour/Bs2PhiNuNu
- flavour/Bc2TauNu
- flavour/B2Kstee
- flavour/Bd2KstNuNu
- flavour/BuBc2TauNu